### PR TITLE
docs: clarify Google add-on versus integration

### PR DESCRIPTION
**Pull request by willnorris**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @willnorris, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;addons#1741).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: google_assistant/README.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ A virtual personal assistant developed by Google.
 
 ## About
 
-[Google Assistant][google-assistant] is an AI-powered voice assistant that runs on the Raspberry Pi and x86 platforms and interact via the [DialogFlow][dialogflow-integration] integration with Home-Assistant. You can also use [Google Actions][google-actions] to extend its functionality.
+[Google Assistant][google-assistant] is an AI-powered voice assistant that runs on the Raspberry Pi and x86 platforms and interact via the [DialogFlow][dialogflow-integration] integration with Home-Assistant.  You can also use [Google Actions][google-actions] to extend its functionality.
```
Comment (frenck) [inline google_assistant/README.md:9]:
```suggestion
[Google Assistant][google-assistant] is an AI-powered voice assistant that runs on the Raspberry Pi and x86 platforms and interact via the [DialogFlow][dialogflow-integration] integration with Home-Assistant. You can also use [Google Actions][google-actions] to extend its functionality.
```

File: google_assistant/README.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ A virtual personal assistant developed by Google.
...
 
-You can simply say "Ok Google" following by your command, and Google Assistant will answer your request.
+This add-on allows you to access Google Assistant using the microphone and speaker attached to the computer or device running Home Assistant.  You can simply say "Ok Google" followed by your command, and Google Assistant will answer your request.
```
Comment (frenck) [inline google_assistant/README.md:11]:
```suggestion
This add-on allows you to access Google Assistant using the microphone and speaker attached to the computer or device running Home Assistant. You can say "Ok Google" followed by your command, and Google Assistant will answer your request.
```

**Review by frenck — APPROVED** [review]
Thanks, @willnorris 👍 

Happy New Year 🥂

**Pull request merged**